packages/devtools_app/lib/src/charts/chart_trace.dart

```diff
@@ -0,0 +1,213 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+import 'chart_controller.dart';
+
+class Data {
+  Data(this.timestamp, this.y);
+
+  final int timestamp;
+  final double y;
+}
+
+class PaintCharacteristics {
+  PaintCharacteristics({
+    this.color = Colors.black,
```

Comment:
defaulting to Colors.black is somewhat at odds with using a color scheme. perhaps make color required?

Reply:
Good idea.